QuickStore/BepInExPlugin.cs

```diff
@@ -105,7 +116,28 @@ private static void StoreItems()
                         if (disallow.Contains(objects[j].GetGroup().GetId()))
                             continue;
                     }
-                    if (!inventory.IsFull() && (ial[i].GetComponent<WorldObjectText>()?.GetText() == Readable.GetGroupName(objects[j].GetGroup()) || inventory.GetInsideWorldObjects().Exists(o => o.GetGroup() == objects[j].GetGroup())))
+                    else if (inventory.IsFull()) continue;
```

Comment:
Not sure if continue is the best option, or break.
Maybe switch to Break? IF the Inventory is full, there i no real reason to proceed the other objects. They would not fit anyway.
But as you made it just skip before, I stayed on this patch for now. Maybe you had a reason.